src/io/state-file/schema.ts

```diff
@@ -301,8 +305,8 @@ const LPSCroppingPlanes: z.ZodType<LPSCroppingPlanes> = z.object({
 const Cropping = z.record(LPSCroppingPlanes);
 
 const Tools = z.object({
-  rulers: Ruler.array(),
-  rectangles: Rectangle.array().optional(),
+  rulers: Rulers.optional(),
+  rectangles: Rectangles.optional(),
```

Comment:
Lesson here is to always make big concept entries Objects, so can extend them with new props (like labels) without breaking backwards compatibility.